docs/rep/controller/openapi.yaml

```diff
@@ -0,0 +1,370 @@
+        '200':
+          description: OK
+          content:
+            text/plain:
+              schema:
+                type: object
+                properties:
+                  tenants:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Tenant'
+  /api/v0/tenants.create:
+    post:
+      tags:
+        - Tenants
+      description: Create a new tenant
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TenantDefinition'
+      responses:
+        '200':
+          description: Successful Response
+  /api/v0/tenants.delete:
+    post:
+      tags:
+        - Tenants
+      description: Delete a tenant
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                id:
+                  type: string
+                  example: "28dcac69-33ef-4b13-a42f-0d07c7acc1a6"
+      responses:
+        '200':
+          description: Successful Response
+  /api/v1/instances.list:
+    get:
+      tags:
+        - Instances
+      description: List all instances
+      parameters:
+        - required: false
+          schema:
+            title: limit
+            minimum: 1
+            type: integer
+          name: limit
+          in: query
+      responses:
+        '200':
+          description: OK
+          content:
+            text/plain:
+              schema:
+                type: object
+                properties:
+                  instances:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Instance'
+          
+  /api/v1/instances.create:
+    post:
+      tags:
+        - Instances
+      description: Create a new instance
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InstanceDefinition'
+      responses:
+        '200':
+          description: Successful Response
+  /api/v0/instance.delete:
+    post:
+      tags:
+        - Instances
+      description: Delete an instance
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                id:
+                  type: string
+                  example: "28dcac69-33ef-4b13-a42f-0d07c7acc1a6"
+      responses:
+        '200':
+          description: Successful Response
+        
+components:
+  schemas:
+  
+    Tenant:   
+      type: object
+      properties:
+        id:
+          type: string
+          example: "f13563b8-8a0b-4867-81f7-6890d6fe9fcd"
+        name:
+          type: string
+          example: <name>
+        verb:
+          type: string
+          example: create
+        resource:
+          type: string
+          example: workload
+    
+    TenantDefinition:   
+      type: object
```

Comment:
For all your schemas, you need to specify which properties are mandatory, by adding a `required` field before specifying the `type`. For example:

```yaml
    WorkloadDefinition:
         required:
         - kind
         - spec
   ```